### ISSUE TITLE
Add an option to disable docker unit tests while building BE app image

### ISSUE THREAD

**Issue by rameshpolishetti** [TIBCOSoftware/be-tools#104]
With the current scripts, docker unit tests are invoked automatically after building the BE app image successfully. Providing an option to disable docker unit tests would help the user to skip tests if he/she wants.

**Comment by tsaxena18** [comment]
Can you list out those unit tests that are executed in the end as part of build process ? (for the record)

**Comment by rameshpolishetti** [comment]
@tsaxena18 Current scripts show the following output (test cases short description + summary how many pass/fail at the end) as unit tests execution summary. Is this the one you are referring?

```sh
appss-MacBook-Pro-8:tests rpolishe$ ./run_tests.sh --image-name s2ibuilderas24ftl64as44:01 --be-version 6.0 --cdd-filename dummy.txt --ear-filename dummy.txt --as-version 4.4 --ftl-version 6.4

=================RUNNING CONTAINER STRUCTURE TESTS=================

-----------------------------------------------
INFO: image name:          [s2ibuilderas24ftl64as44:01]
INFO: be version:          [6.0]
INFO: cdd file name:       [fd.cdd]
INFO: ear file name:       [fd.ear]
INFO: as version:          [4.4]
INFO: ftl version:         [6.4]
-----------------------------------------------


================================
====== Test file: be.yaml ======
================================

================================
====== Test file: as.yaml ======
================================

=================================
====== Test file: ftl.yaml ======
=================================
=== RUN: Command Test: beengine without cdd
--- PASS
duration: 6.3628339s
=== RUN: File Content Test: BE TRA Updated with AS home
--- PASS
duration: 0s
=== RUN: File Existence Test: BE Path
--- PASS
duration: 0s
=== RUN: File Existence Test: BE Engine Binary
--- PASS
duration: 0s
=== RUN: File Existence Test: BE Engine TRA File
--- PASS
duration: 0s
=== RUN: File Existence Test: BE Path Lib
--- PASS
duration: 0s
=== RUN: File Existence Test: BE Lib File
--- PASS
duration: 0s
=== RUN: File Existence Test: Gvproviders Path
--- PASS
duration: 0s
=== RUN: File Existence Test: Gvproviders File
--- PASS
duration: 0s
=== RUN: File Existence Test: Gvproviders Consul
--- PASS
duration: 0s
=== RUN: File Existence Test: Gvproviders Consul File
--- PASS
duration: 0s
=== RUN: File Existence Test: Docker Run File
--- PASS
duration: 0s
=== RUN: File Existence Test: Cdd file
--- PASS
duration: 0s
=== RUN: File Existence Test: Ear file
--- PASS
duration: 0s
=== RUN: File Existence Test: Cdd file
--- PASS
duration: 0s
=== RUN: File Existence Test: Ear file
--- PASS
duration: 0s
=== RUN: Metadata Test
--- PASS
duration: 0s
=== RUN: File Content Test: BE TRA Updated with AS home
--- PASS
duration: 0s
=== RUN: File Existence Test: AS Lib Path
--- PASS
duration: 0s
=== RUN: File Existence Test: AS Lib Jar File
--- PASS
duration: 0s
=== RUN: File Content Test: BE TRA Updated with FTL home
--- PASS
duration: 0s
=== RUN: File Existence Test: FTL Lib Path
--- PASS
duration: 0s
=== RUN: File Existence Test: FTL Lib Jar File
--- PASS
duration: 0s

=================================
============ RESULTS ============
=================================
Passes:      23
Failures:    0
Duration:    6.3628339s
Total tests: 23

PASS

=================END OF CONTAINER STRUCTURE TESTS==================
```

**Comment by tsaxena18** [comment]
Yes, so as a part of image creation what are the things that we are checking here. From summary I can deduce that we are checking the application and environment files. Anything else ?

**Comment by rameshpolishetti** [comment]
Docker unit tests validate the structure of the BE Application container image. Which includes the following validations on the BE app image:

1. Output of important commands
2. metadata of the image
3. Whether important files are present
4. Whether important files are updated correctly

Detailed information is available at - https://github.com/TIBCOSoftware/be-tools/tree/master/cloud/docker/tests

**Comment by tsaxena18** [comment]
Its fine. I have already verified the issue on pr#105. We can close this.

**Comment by rameshpolishetti** [comment]
closing this as it is implemented and merged into master through the PR #105
